.github/workflows/translations.yaml

```diff
@@ -0,0 +1,57 @@
+name: Translations
+
+on:
+  push:
+    branches:
+      - main
+      - 'releases/**'
+  pull_request:
+    branches:
+      - main
+      - 'releases/**'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  translations:
+    name: Translations
+    runs-on: ubuntu-latest
+    env:
+      QTVERSION: 6.2.4
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v3
+
+      - name: Checkout submodules
```

Comment:
BTW, do you really need to update submodules? I assume there are no strings in there, but I might be wrong. We definitely don't do it in the l10n action.